kim/kim-ldap/src/main/java/edu/arizona/kim/service/impl/UaEdsIdentityService.java

```diff
@@ -27,6 +36,12 @@
 public class UaEdsIdentityService extends LdapIdentityDelegateServiceImpl implements LdapIdentityService {
```

Comment:
This is not an interface, but an actual implementation and there is no EDS specific code in here so the name should be UALdapIdentityServiceImpl. Also it should replace completely LdapIdentityDelegateServiceImpl in all the spring wiring and as such, it should implement directly LdapIdentityService instead of extending  LdapIdentityDelegateServiceImpl.
Scott mentioned this would take a lot of testing effort and we don't have the necessary resources right now, as such we will be using this temporary solution for now.